config/suit/templates/nrf54h20/default/v1/root_with_binary_nordic_top.yaml.jinja2

```diff
@@ -1,3 +1,14 @@
+# This SUIT manifest template can be for a variety of scenarios,
```

Comment:
This SUIT manifest template can be used in various scenarios?

Reply:
Fixed